tabs-tabs-tabs/tabs.js

```diff
@@ -122,5 +122,12 @@ document.addEventListener("click", function(e) {
     });
   }
 
+  else if (e.target.id === "tabs-highlight") { // highlights current tab and next tab
```

Comment:
Oh yeah I noticed that, but wasn't sure what to do about it; I'll add the comment in!